src/main/java/killrvideo/graph/KillrVideoTraversalSourceDsl.java

```diff
@@ -0,0 +1,164 @@
+package killrvideo.graph;
```

Comment:
This is a really neat question which forced me to think about how to explain what is going on.  I am using a recommended DSL implementation pattern which is explained in detail here -> https://www.datastax.com/dev/blog/gremlin-dsls-in-java-with-dse-graph, but for lighter reading 
I'll give you a snippet.
<img width="1086" alt="screen shot 2017-08-29 at 2 00 28 pm" src="https://user-images.githubusercontent.com/23346205/29836197-7d5adde4-8cc2-11e7-96e6-7c4606ca0574.png">

Honestly, I could probably do a whole blog post on this topic alone.

So, if you are just looking at the KillrVideoTraversalDSL interface and the KillrVideoTraversalSourceDSL class you see here in the pull request it might not seem obvious what is going on and what the difference between them is.  In short, these are both part of a recommended pattern using a GremlinDsl annotation to generate boilerplate code for the underlying implementation.  The interface defines the step methods (e.g out(), in(), select(), etc) for the graph traversal DSL and the class spawns KillrVideoTraversalSource instances which sit atop of GraphTraversalSource and START our DSL traversals.  We are effectively slapping our KillrVideo specific language on top of the default g.V()... gremlin traversals.  So from g.V().hasLabel("video").has("videoId", "someIdHere") to killr.videos("someIdHere").  Also, because our traversals START from those defined in our class it is always assumed you are coming "through" whatever method is defined in the class when invoking those defined in the interface.